tests/test_update.js

```diff
@@ -1,54 +1,62 @@
 /*
 Testing record creation.
 */
-const Zotero = require('../build/zotero-lib')
-const fs = require('fs')
+const Zotero = require('../build/zotero-lib');
 
 async function main() {
-  var zotero = new Zotero({ verbose: false, "group-id": 2259720 })
+  var zotero = new Zotero({ verbose: false, 'group-id': 2259720 });
   const template = {
-    template: "report"
-  }
-  let report = await zotero.create_item(template)
-  report.title = "Title here"
+    template: 'report',
+  };
+  let report = await zotero.create_item(template);
+  report.title = 'Title here';
   const zoteroResult = await zotero.create_item({
     item: report,
-    fullresponse: true
-  })
-  const zoteroRecordVersion = zoteroResult.successful["0"].version
-  const zoteroRecordGType = zoteroResult.successful["0"].library.type
-  const zoteroRecordGroup = zoteroResult.successful["0"].library.id
-  const zoteroItem = zotero.pruneData(zoteroResult)
-  // Now update the zenodo record with the ZoteroId.
-  if (zoteroRecordGType != "group") {
-    return this.message(1, "ERROR: group-type=user not implemented")
+    fullresponse: true,
+  });
+  const zoteroRecordVersion = zoteroResult.successful['0'].version;
+  const zoteroRecordGType = zoteroResult.successful['0'].library.type;
+  const zoteroItem = zotero.pruneData(zoteroResult);
+  if (zoteroRecordGType !== 'group') {
+    return this.message(1, 'ERROR: group-type=user not implemented');
   }
   const args = {
     key: zoteroItem.key,
     version: zoteroRecordVersion,
-    update: { url: "https://kerko_url" },
+    update: { url: 'https://kerko_url' },
+    json: {
+      title: 'updated title',
+    },
     fullresponse: false,
-    show: true
-  }
-  console.log("--------------------------------")
-  const update = await zotero.update_item(args)
+    show: true,
+  };
+  console.log('--------------------------------');
```

Comment:
Kindly remove the console.log. would be better to add it in eslint to alert you about it too.